WooCommerce/Classes/ViewRelated/Dashboard/Settings/SettingsViewController.swift

```diff
@@ -216,6 +217,15 @@ private extension SettingsViewController {
         present(alertController, animated: true)
     }
 
+    func changeStoreWasPressed() {
+        StoresManager.shared.resetAuthentication()
+
```

Comment:
@mindgraffiti I still have too test this locally, but this is the point where I am thinking we should:
1. Clear all the orders via `OrderAction`resetStoredOrders()`
2. Clear all of the stats data via a **new** action in `StatsAction.resetStoredStats()`

After this we can then show the site picker proceed normally.